src/ucs/datastruct/callbackq.c

```diff
@@ -403,7 +403,7 @@ ucs_status_t ucs_callbackq_init(ucs_callbackq_t *cbq)
     ucs_callbackq_priv_t *priv = ucs_callbackq_priv(cbq);
     unsigned idx;
 
-    for (idx = 0; idx < UCS_CALLBACKQ_FAST_COUNT; ++idx) {
+    for (idx = 0; idx < UCS_CALLBACKQ_FAST_COUNT + 1; ++idx) {
```

Comment:
It's not clear for me how decrementing of the constant and incrementing of the loop condition fixes overflow. Moreover, 
`#define UCS_CALLBACKQ_FAST_MAX       (UCS_CALLBACKQ_FAST_COUNT - 1)`
@yosefe could you clarify which case fixes the patch?